### PR TITLE
Reformat 10 recipes

### PR DESCRIPTION
**Pull request by DarkmatterVale**
This PR reformats 10 recipes to bring them up to current standards (https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines).

GCI task: https://codein.withgoogle.com/dashboard/task-instances/5725172489256960/?sp-page=1

### CODE REVIEW

**Comment by korli** [comment]
Wouldn't it make sense to write a script to automate such a reformat?

**Comment by waddlesplash** [comment]
> Wouldn't it make sense to write a script to automate such a reformat?

Such a script would have to parse & understand all of Bash syntax, and I was unable to find anything that did so in the past.

File: haiku-apps/artpaint/artpaint-2.1.0git.recipe [hunk 1]
```diff
@@ -5,13 +5,16 @@ The key design principle has been to let the user see what they are doing \
 instead of just forcing them to type in some numbers and then hoping that \
 the result will not be too far from the expected."
 HOMEPAGE="https://github.com/HaikuArchives/ArtPaint"
+COPYRIGHT="
+	2003 Heikki Suhonen
+	2009 Karsten Heimrich
+	"
```
Comment (waddlesplash) [inline haiku-apps/artpaint/artpaint-2.1.0git.recipe]: COPYRIGHT was formatted the correct way before (the way it is below).

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**